bundle-rest-embedded/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'net.corda.plugins.cordapp'
 
 cordapp {
     targetPlatformVersion 7
-    minimumPlatformVersion 7
+    minimumPlatformVersion 6
```

Comment:
Worth adding a comment explaining why it's the minimum version

Reply:
Updated commit message